id/proguard-rules.pro

```diff
@@ -89,20 +37,6 @@
 -keepclassmembernames class kotlin.coroutines.SafeContinuation {
     volatile <fields>;
 }
-# https://github.com/Kotlin/kotlinx.coroutines/issues/1270
--dontwarn kotlinx.coroutines.flow.**
--dontwarn com.simprints.id.services.scheduledSync.people.up.workers.**
--dontwarn com.simprints.id.tools.extensions.**
--dontwarn com.simprints.id.activities.settings.syncinformation.**
--dontwarn com.simprints.id.activities.consent.**
--dontwarn com.simprints.infra.events.session.**
-
-# https://github.com/Kotlin/kotlinx.coroutines/issues/2046
--dontwarn java.lang.instrument.ClassFileTransformer
--dontwarn sun.misc.SignalHandler
--dontwarn java.lang.instrument.Instrumentation
--dontwarn sun.misc.Signal
```

Comment:
All the dontwarn's are not needed?

Reply:
Did not see any issues due to this removal 🤷🏻